### ISSUE TITLE
Generate other libraries for exporting game to other platforms (like Linux)

### ISSUE THREAD

**Issue by DARTHVADERbn** [pragmagic/godot-nim#104]
When i trying to export game to Linux or Windows 32 bits 
i'm getting error: can't find library `{name_of_library}`

Example: Trying to export game to Windows 32 bits
`core/os/dir_access.cpp:297 - Failed to open res://_dlls/nim_32.dll`

**Comment by Knaque** [comment]
Are you trying to export for Linux/Win32 *from* Win64?

**Comment by DarthvaderBn** [comment]
Yes, but i realized what `godot-nim` generates .dll only for your OS
https://github.com/geekrelief/godot-nim-stub/blob/master/nakefile.nim

**Comment by Knaque** [comment]
That's what I was going to point out to you. I'm glad you found it on your own. In theory, one could set things up to allow for cross-compiling, but by default godot-nim only lets you compile for the current platform.
